zer0-os/zOS
# feat: add zApp NFTS to app container

### What does this do?
- Adds a new external package: [`@zero-tech/zapp-nfts`](https://www.npmjs.com/package/@zero-tech/zapp-nfts) - this is the nfts zApp.
- Adds the App.NFTS enum value to the list of active apps in lib/apps/index.ts.
- Renders NFTS zApp when selectedApp === App.NFTS.

Short answer: it adds zApp NFTS to the list of apps.

### Why are we making this change?
- To break ground on the relationship between zApps and zOS - this PR will act as validation of our process.

### How do I test this?
- Click "NFTS" in the app selector.
- Check that the "NFTS" app renders.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
